### PR TITLE
Add calendar picker on tasks pages

### LINKED ISSUE #3: Use Redux

### LINKED ISSUE #43: Add calendar picker on task list
A calendar should open when clicking on the date in `DateSelectHeader`, for example.

### PR DESCRIPTION
**Pull request by Atala**
Fix #43 
Fix #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,17 +8,24 @@
   },
   "dependencies": {
     "@mapbox/polyline": "^0.2.0",
+    "metro": "^0.26.0",
```
Comment (alexsegura) [inline package.json]: Is this needed? It should be required by `react-native`.

File: src/page/courier/TaskPage.js [hunk 2]
```diff
@@ -243,7 +218,7 @@ class TaskPage extends Component {
 
     const swipeoutLeftButton = {
       component: this.renderSwipeoutLeftButton(),
```
Comment (alexsegura) [inline src/page/courier/TaskPage.js]: Error when opening a task: `this.renderSwipeoutLeftButton is not a function`

**Comment by alexsegura** [comment]
Redux ALL THE THINGS 🙂

Just one thought: the button to open the calendar is small and very close to the arrow, it is hard to click on it on a real device (see screenshot below when removing `transparent` property). 

I think it would be better to toggle the calendar by tapping on the date. 

capture d ecran 2018-02-24 a 03 10 07

File: package.json (hunk 1 above)
Comment (Atala) [reply package.json]: Yes sorry, I'll remove it. I had issue with the `metro ` cache.

**Comment by Atala** [comment]
Ok for this change. Do you think it is intuitive enough that one can click on the date to open the calendar? I think yes.

**Comment by alexsegura** [comment]
I think yes ✅

Also, the arrow buttons should take all the remaining space available.

**Comment by Atala** [comment]
@alexsegura I implemented all the changes you requested?

Can you check and merge if it's ok for you, so Elias can start the websocket stuff?

**Comment by couleurmate** [comment]
Saw it late, but I think it's not fundamental to have a calendar picker per se, the arrows are enough. However, it would be more intuitive to click on the date, at least that what's done on most calendar app on Android.

**Pull request merged**